docs/source/asr/sc/datasets.rst

```diff
@@ -0,0 +1,123 @@
+You can get started with those datasets by following the instructions to run those scripts in the section appropriate to each dataset below.
+
+If you have your own data and want to preprocess it to use with NeMo ASR models, check out the `Preparing Custom Speech Classification Data`_ section at the bottom of the page.
+
+.. _Freesound-dataset:
+
+Freesound
+-----------
+
+`Freesound <http://www.freesound.org/>`_ is a website that aims to create a huge open collaborative database of audio snippets, samples, recordings, bleeps. 
+Audio samples are released under Creative Commons licenses that allow their reuse. 
+Researchers and developers can access Freesound content using the Freesound API to retrieve meaningful sound information such as metadata, analysis files, and the sounds themselves. 
+
+**Instructions**
+
+Go to ``<NeMo_git_root>/scripts/freesound_download_resample`` and follow the below steps to download and convert freedsound data into a format expected by the `nemo_asr` collection.
+
+1. We will need some requirements including freesound, requests, requests_oauthlib, joblib, librosa and sox. If they are not installed, please run `pip install -r freesound_requirements.txt`
+2. Create an API key for freesound.org at https://freesound.org/help/developers/
+3. Create a python file called `freesound_private_apikey.py` and add lined `api_key = <your Freesound api key> and client_id = <your Freesound client id>`
+4. Authorize by run `python freesound_download.py --authorize` and visit the website and paste response code
+5. Feel free to change any arguments in `download_resample_freesound.sh` such as max_samples and max_filesize
+6. Run `bash download_resample_freesound.sh <numbers of files you want> <download data directory> <resampled data directory>` . For example: 
+
+.. code-block:: bash
+
+    bash download_resample_freesound.sh 4000 ./freesound ./freesound_resampled_background
+
+Note that downloading this dataset may take hours. Change categories in download_resample_freesound.sh to include other (speech) categories audio files.
+Then, you should have 16khz mono wav files in `<resampled data directory>`. 
+
+
+.. _Google-Speech-Commands-Dataset:
+
+Google Speech Commands Dataset
+------------------------------
+
+Google released two versions of the dataset with the first version containing 65k samples over 30 classes and the second containing 110k samples over 35 classes.
+We refer to these datasets as v1 and v2
+
+Run the script `process_speech_commands_data.py` to process Google Speech Commands dataset in order to generate files in the supported format of  `nemo_asr`,
+which can be found in ``<NeMo_git_root>/scripts/dataset_processing/``. 
+You should set the data folder of Speech Commands using :code:`--data_root` and the version of the dataset using :code:`--data_version` as an int.
+
+You can further rebalance the train set by passing the `--rebalance` flag.
```

Comment:
`rebalance the train set by randomly oversampling files inside the manifest by passing the ...`